config/manifests/preview/secrets-store-csi-driver-operator.clusterserviceversion.yaml

```diff
@@ -14,6 +14,16 @@ metadata:
     createdAt: "2023-06-12T00:00:00Z"
     description: Install and configure Secrets Store CSI driver.
     olm.skipRange: ">=4.13.0-0 <4.16.0"
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "true"
+    features.operators.openshift.io/proxy-aware: "true"
+    features.operators.openshift.io/tls-profiles: "true"
```

Comment:
Is this correct? The doc says it's "a way to regulate which ciphers a client can use": https://docs.openshift.com/container-platform/4.14/security/tls-security-profiles.html

My understanding is that this is the TLS Ciper feature Fabio was working on recently. I can see it in EFS operator, but not LSO nor secret store operator.

Reply:
Doh, you're right, updated. Thanks :)